src/playground/src/ui/screen/checkbox.component.tsx

```diff
@@ -29,26 +38,102 @@ class CheckBox extends React.Component<Props, State> {
   };
 
   public state: State = {
-    isCheckBox1Checked: false,
-    isCheckBox2Checked: true,
-    isCheckBox3Checked: false,
-    isCheckBox4Checked: true,
+    isCheckBoxPrimaryChecked: true,
+    isCheckBoxSuccessChecked: true,
+    isCheckBoxWarningChecked: true,
+    isCheckBoxDangerChecked: true,
+    isCheckBoxInfoChecked: true,
+    isCheckBoxWhiteChecked: true,
+    isCheckBoxDisabled1Checked: true,
+    isCheckBoxDisabled2Checked: false,
+    isIndeterminate: false,
+    indeterminateValue: undefined,
+    treeCheckbox1Value: true,
+    treeCheckbox2Value: false,
+    treeCheckbox3Value: false,
   };
 
-  private onCheckBox1Change = (selected: boolean) => {
-    this.setState({ isCheckBox1Checked: selected });
+  public componentWillMount(): void {
+    this.changeIndeterminate();
+  }
+
+  private onCheckBoxPrimaryChange = (selected: boolean) => {
+    this.setState({ isCheckBoxPrimaryChecked: selected });
+  };
+
+  private onCheckBoxSuccessChange = (selected: boolean) => {
+    this.setState({ isCheckBoxSuccessChecked: selected });
   };
 
-  private onCheckBox2Change = (selected: boolean) => {
-    this.setState({ isCheckBox2Checked: selected });
+  private onCheckBoxWarningChange = (selected: boolean) => {
+    this.setState({ isCheckBoxWarningChecked: selected });
   };
 
-  private onCheckBox3Change = (selected: boolean) => {
-    this.setState({ isCheckBox3Checked: selected });
+  private onCheckBoxDangerChange = (selected: boolean) => {
+    this.setState({ isCheckBoxDangerChecked: selected });
   };
 
-  private onCheckBox4Change = (selected: boolean) => {
-    this.setState({ isCheckBox4Checked: selected });
+  private onCheckBoxInfoChange = (selected: boolean) => {
+    this.setState({ isCheckBoxInfoChecked: selected });
+  };
+
+  private onCheckBoxWhiteChange = (selected: boolean) => {
+    this.setState({ isCheckBoxWhiteChecked: selected });
+  };
+
+  private onCheckBoxDisabled1Change = (selected: boolean) => {
+    this.setState({ isCheckBoxDisabled1Checked: selected });
+  };
+
+  private onCheckBoxDisabled2Change = (selected: boolean) => {
+    this.setState({ isCheckBoxDisabled2Checked: selected });
+  };
+
+  private onIndeterminateValue = (selected: boolean) => {
```

Comment:
I like this example, but
I don't think we need it to be hardcoded in playground
Prefer refactoring to separated component, lets say `CheckList` to demonstrate this

Reply:
But it won't  be part of a framework. There is no time to transfer to a separate component. In any case, this can be part of the documentation as one example of using the checkbox.